### PR TITLE
feat: todo example

### PR DESCRIPTION
**Pull request by ethanve**

### CODE REVIEW

File: examples/todo/.eslintignore [hunk 1]
```diff
@@ -0,0 +1,5 @@
+node_modules
```
Comment (taion) [inline examples/todo/.eslintignore:1]: could maybe use the same one as in base. or even just have the base package handle linting here

File: examples/todo/.gitignore [hunk 2]
```diff
@@ -0,0 +1,4 @@
+.DS_Store
```
Comment (taion) [inline examples/todo/.gitignore:1]: maybe not needed? shouldn't the base gitignore already handle this stuff?

File: examples/todo/src/components/TodoApp.tsx [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+function TodoApp({ relay, user }: Props) {
+  const handleTextInputSave = (text: string) => {
+    AddTodoMutation.commit(relay.environment, text, user);
+  };
+
+  const hasTodos = user.totalCount > 0;
+
+  useEffect(() => {
+    // initialize subscriptions
+    const removeDisposable = removeTodoSubscription();
```
Comment (taion) [inline examples/todo/src/components/TodoApp.tsx:39]: might be better to name this as `requestRemoveTodoSubscription`, &c.

**Comment by ethanve** [comment]
Really just copy/pasted the relay todo example, diffed and copied our flow version, then just got it to work with relay-network-layer. So basically what you said 😆

**Comment by jquense** [comment]
I SEE YOU ETHAN

**Comment by taion** [comment]
Why didn't we just merge this again?

**Comment by jquense** [comment]
I don't know 👋  hey Jimmy

**Comment by jquense** [comment]
tbh i didn't realize this was here I would have used it in #407 where I added half of this

**Pull request closed without merging**